### PR TITLE
Add autocorrect to internal affairs cops

### PR DESCRIPTION
**Pull request by rrosenblum**

### CODE REVIEW

**Comment by Drenmi** [comment]
Looks like I didn't rename the node matcher in `RedundantMessageArgument`. Feel free to fix it to something better as part of this change. 😅

**Comment by bbatsov** [comment]
You've got a typo in the first commit message.

File: spec/rubocop/cop/internal_affairs/node_type_predicate_spec.rb [hunk 1]
```diff
@@ -3,11 +3,19 @@
...
+  context 'comparison node type check' do
+    it 'registers an offense' do
+      expect_offense(<<-RUBY, 'example_cop.rb')
       node.type == :send
       ^^^^^^^^^^^^^^^^^^ Use `#send_type?` to check node type.
-    RUBY
+      RUBY
+    end
+
+    it 'corrects' do
```
Comment (bbatsov) [inline spec/rubocop/cop/internal_affairs/node_type_predicate_spec.rb]: In the other commit you've used the description "auto-corrects", which I prefer.

**Comment by bbatsov** [comment]
You still have a typo in the first comment message - two typos to be precise.

**Comment by rrosenblum** [comment]
Weird, I swear I fixed the typos a few days ago. Oh well, they have been fixed now.

**Pull request merged**